src/metabase/driver/query_processor.clj

```diff
@@ -92,6 +92,12 @@
                                                   msg)))
                                             explanation))))
 
+
+;; +-------------------------------------------------------------------------------------------------------------------+
```

Comment:
the effect is that the code follows conventions and idioms and is formatted properly